src/components/Form/SubmitButton.styled.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export const SubmitButton = styled.button`
+	display: flex;
+	flex-direction: column;
```

Comment:
Is `flex-direction: column` really necessary? 🤔

Reply:
Changed

Reply:
changed